### ISSUE TITLE
Investigate how assistive technologies interact with character count 

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-frontend#2539]
## What
We're looking to investigate how screen reader assistive technologies interact with the character count component.

https://design-system.service.gov.uk/components/character-count/
https://govuk-frontend-review.herokuapp.com/components/character-count

The investigation will focus primarily on gathering information to help resolve four related bugs, listed below.

### Related bug tickets
This investigation will help us understand the bugs listed in the four tickets below.

- https://github.com/alphagov/govuk-frontend/issues/2485
- https://github.com/alphagov/govuk-frontend/issues/2486
- https://github.com/alphagov/govuk-frontend/issues/2487
- https://github.com/alphagov/govuk-frontend/issues/2488

## Why
The character count component was initially designed to play well with screen readers, but sometime after the launch of the component something changed. Now there are four bugs that make interacting with the character count component via 
 a screen reader frustrating and cumbersome.

This might be an accessibility compliance issue, but more investigation would be needed to confirm. For now, we're approaching it as a severe usability issue and poor experience for those using a screen reader (and potentially other assistive technologies as well).

## Who needs to work on this

- @davidc-gds will handle the accessibility portions of the investigation and test with various browser and assistive technology combinations.
- @owenatgov has offered to support from a developer perspective if capacity allows, since the character count component includes some JavaScript.

## Who needs to review this
Unknown

## Done when

- [ ] Character count component variants are tested on various screen readers and browsers
- [ ] Research is compiled into a summary
- [ ] All 4 identified bugs have been explored
- [ ] Action items and approaches are identified and proposed for each of the four identified bugs

**Comment by 36degrees** [comment]
@davidc-gds @querkmachine is there anything else to do here or can we call this one done?

**Comment by querkmachine** [comment]
@36degrees We're still waiting on the guidance in the design system to be updated to reflect the changes made. We kept this issue open to accommodate for that but having a new issue in that repo might make more sense? @davidc-gds

**Comment by vanitabarrett** [comment]
@querkmachine A new issue in the Design System repo would make sense to me - we could attach it to the v4.1.0 milestone then too

**Comment by davidc-gds** [comment]
I've set up the new issue for the guidance content updates. Feel free to edit or modify it if I've built it incorrectly:
https://github.com/alphagov/govuk-design-system/issues/2155

@querkmachine would this mean that this issue can now be closed, and we'll hop over to that new issue?

**Comment by querkmachine** [comment]
Sounds like a plan!
